test/common.dart

```diff
@@ -295,6 +297,16 @@ main() {
       _checkQuery(res, [#b, #d, #f, #g, #h, #i]);
     });
 
+    test('overriden symbols', () {
+      var options = new smoke.QueryOptions(
+          excludeOverriden: true, includeInherited: true, includeMethods: true);
+      var res = smoke.query(L2, options);
+      _checkQuery(res, [#m, #incM, #n]);
+      // Check that the concrete #j is there
+      expect(res[0].isFinal, false);
```

Comment:
haha - seems like a bug in toString :-), we say "method" even when it's a field.

I think what's happening here is that in the static config the order of the result might be different than in the mirror implementation. I think the mirror implementation returns things in the order they are declared, but in the static config we might be returning things in the order we put then in the table (which is sorted alphabetically), so it's possible that res[0] is always `#incM`.